axonserver/src/main/java/io/axoniq/axonserver/config/MessagingPlatformConfiguration.java

```diff
@@ -225,6 +239,13 @@ public SslConfiguration getSsl() {
     }
 
     public void setSsl(SslConfiguration ssl) {
+        if (ssl == null) {
```

Comment:
I get the if-else block here to reach the right point to log yes/no, but wouldn't it be nicer to allow the `AuditLogger` to receive a predicate of some kind to let it decide when to log? 
What I am essentially looking for is to not have this class "obstructed" with logging decision making logic like this.

Reply:
My thoughts were that the security related configuration settings are important, and I want it publicly shouted out what Axon Server is using and how it changes. I could make this into a single method with old value, new value, name for setting, name for removing, name for being enabled, name for being disabled, and name of the setting itself, but this is IMHO simpler.